### ISSUE TITLE
false positive, repid not found

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3608]
```abap
DEFINE _foo.
WRITE / sy-repid.
END-OF-DEFINITION.

_foo.
```
